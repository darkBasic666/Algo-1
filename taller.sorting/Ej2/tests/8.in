8 0
[(Chaco;10000),(Salta;129312),(Corrientes;10000),(Neuquen;129387),(Cordoba;10000),(Santiago;500),(Viedma;32343),(Catamarca;100000)]