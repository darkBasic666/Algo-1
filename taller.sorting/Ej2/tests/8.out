8 0
[(Santiago;500),(Chaco;10000),(Cordoba;10000),(Corrientes;10000),(Viedma;32343),(Catamarca;100000),(Salta;129312),(Neuquen;129387)]
